scheduler/config/config.go

```diff
@@ -639,23 +635,23 @@ func (cfg *Config) Validate() error {
 		}
 	}
 
-	if cfg.NetworkTopology.CollectInterval <= 0 {
+	if cfg.Scheduler.NetworkTopology.CollectInterval <= 0 {
```

Comment:
Condition cfg.Scheduler.Algorithm equals evaluator.NetworkTopologyAlgorithm must be met to check `NetworkTopology` config.